components/freeswitch/Dockerfile

```diff
@@ -1,3 +1,28 @@
+FROM public.ecr.aws/docker/library/debian:bookworm-slim as rayo-build
```

Comment:
I think we can merge this into the existing `freeswitch-build` build stage. The purpose of this throw away stage is to handle all the native builds that we need for FreeSWITCH